state_management/javascript/sdk/order-processor/index.js

```diff
@@ -1,45 +1,41 @@
-import { DaprClient } from 'dapr-client'; 
+import { DaprClient } from 'dapr-client'
```

Comment:
Please use `@dapr/dapr` instead of `dapr-client` as the latter is being deprecated

Reply:
Changed to use `@dapr/dapr`